services-custom/dynamodb-enhanced/src/it/java/software/amazon/awssdk/enhanced/dynamodb/ScanQueryIntegrationTest.java

```diff
@@ -57,6 +63,60 @@ public static void setup() {
         mappedTable = enhancedClient.table(TABLE_NAME, TABLE_SCHEMA);
         mappedTable.createTable();
         dynamoDbClient.waiter().waitUntilTableExists(r -> r.tableName(TABLE_NAME));
+
+        // sandbox
+
+        QueryConditional queryConditional = QueryConditional.keyEqualTo(Key.builder().partitionValue("val").build());
+
+        PageIterable<Record> pageIterable = mappedTable.query(QueryEnhancedRequest.builder()
+                                                                                  .queryConditional(queryConditional)
+                                                                                  .select(Select.COUNT)
+                                                                                  .build());
+        Iterator<Page<Record>> iterator = pageIterable.iterator();
+
+        Page<Record> page = iterator.next();
+
+        int count = page.count();
+    }
+
+    public class NullifyEmptyStringConverter implements AttributeConverter<String> {
```

Comment:
Why do we need this ?

Reply:
oops unrelated accidental commit, will remove